cmd/cloud-controller-manager/BUILD

```diff
@@ -76,3 +76,75 @@ go_test(
         "//vendor/k8s.io/controller-manager/app",
     ],
 )
+
+go_cross_binary(
+    name = "bin_amd64",
+    platform = "@io_bazel_rules_go//go/toolchain:linux_amd64",
+    target = ":cloud-controller-manager_bin",
+)
+
+go_cross_binary(
+    name = "bin_arm64",
+    platform = "@io_bazel_rules_go//go/toolchain:linux_arm64",
+    target = ":cloud-controller-manager_bin",
+)
+
+pkg_tar(
+    name = "layer_amd64",
+    srcs = [":bin_amd64"],
+    package_dir = "/usr/local/bin",
+)
+
+pkg_tar(
+    name = "layer_arm64",
+    srcs = [":bin_arm64"],
+    package_dir = "/usr/local/bin",
+)
+
+oci_image(
+    name = "image_amd64",
+    entrypoint = ["/usr/local/bin/bin_amd64"],
+    tars = [":layer_amd64"],
+    os = "linux",
+    architecture = "amd64",
+)
+
+oci_image(
+    name = "image_arm64",
+    entrypoint = ["/usr/local/bin/bin_arm64"],
+    tars = [":layer_arm64"],
+    os = "linux",
+    architecture = "arm64",
+)
+
+oci_load(
+    name = "load_amd64",
+    image = ":image_amd64",
+    repo_tags = ["my-repo/cloud-controller-manager:local"],
+)
+
+oci_load(
+    name = "load_arm64",
+    image = ":image_arm64",
+    repo_tags = ["my-repo/cloud-controller-manager:local"],
+)
+
+oci_image_index(
+    name = "cloud-controller-manager_index",
+    images = [
+        ":image_amd64",
+        ":image_arm64",
+    ],
+)
+
+oci_push(
+    name = "push_multiplatform_to_gcr",
+    image = ":cloud-controller-manager_index",
+    repository = "{repository_placeholder}",
+)
+
+# Legacy single platform image
```

Comment:
Hey, I authored the PR you referenced, we don't use bazel to build the images so you can remove this section.